libs/sdk-react-native/src/index.tsx

```diff
@@ -577,3 +581,8 @@ export const recommendedFees = async (): Promise<RecommendedFees> => {
     const response = await BreezSDK.recommendedFees()
     return response as RecommendedFees
 }
+
+export const buyBitcoin = async (provider: BuyBitcoinProvider): Promise<string> => {
+    const response = await BreezSDK.buyBitcoin(provider)
```

Comment:
This call should be implemented in both Android and iOS native modules:
BreezSDKModule.kt
RNBreezSDK.swift

Reply:
Just added.

Reply:
Needs this adding to the bottom of `ios/RNBreezSDK.m`
```
RCT_EXTERN_METHOD(
    buyBitcoin: (NSString*)provider
    resolver: (RCTPromiseResolveBlock)resolve
    rejecter: (RCTPromiseRejectBlock)reject
)
```